custom_components/jcihitachi_tw/climate.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, thing, coordinator):
         self._supported_fan_modes = [fan_mode for i, fan_mode in enumerate(SUPPORT_FAN) if 2 ** i & self._thing.support_code.FanSpeed != 0]
         self._supported_hvac = [SUPPORT_HVAC[0]] + [hvac for i, hvac in enumerate(SUPPORT_HVAC[1:]) if 2 ** i & self._thing.support_code.Mode != 0]
         self._supported_presets = self.calculate_supported_presets()
+        self._prev_target = -1
```

Comment:
`self._thing.support_code.min_temp` can be used to initialize it.

Reply:
Thanks for the hint, I had modified the line in new commit